### DISCUSSION TITLE
The ability to execute RCON and Admin command right from the app.

### DISCUSSION THREAD

**Discussion by SupremeJacket** [seDirector/App#16] [Ideas]
I think it would be great if we can do this without the need to show/hide the server console, cause often it's bombarded with a lot of different (sometime useless) information. Even better with a separate panel on the main page. Example with Source Admin Tool:
![image](https://user-images.githubusercontent.com/66083398/136496795-1ab5a271-753a-4e56-be1f-5819908d4c7c.png)

**Comment by asheroto** [comment]
Great idea.  Added to planned features: https://github.com/seDirector/App/projects/2#card-87416287

**Reply by SupremeJacket** [reply]
Nice! Looking forward to it!

**Comment by Nomy61** [comment]
that would be awesome
